freqtrade/exchange.py

```diff
@@ -2,18 +2,23 @@
 import logging
```

Comment:
I think we should remove this file and put the logic in `exchanges/__init__.py`

Reply:
This causes a circular import in the bittrex module for the Exchange class. I would suggest moving the interface to e.g. exchange/interface.py. This should work.

Reply:
Ah ok I see, yes this should work. Another solution would be to use a inline import, both options are fine for me

Reply:
Went for a new module, it should be cleaner